framework/auth/oauth_scopes.py

```diff
@@ -192,6 +192,13 @@ class CoreScopes(object):
     READ_REGISTRATION_RESOURCES = 'read_registration_resources'
     WRITE_REGISTRATION_RESOURCES = 'write_registration_resources'
 
+    READ_COLLECTION_SUBMISSION_ACTION = 'read_collection_submission_action'
+    WRITE_COLLECTION_SUBMISSION_ACTION = 'write_collection_submission_action'
```

Comment:
I don't think we need these. We use `[READ/WRITE]_SCHEMA_RESPONSE` as our scope for `SchemaResponseActionList`, for example.